spec/controllers/your_details_controller_spec.rb

```diff
@@ -70,34 +70,30 @@ module Wpcc
 
     describe '#create' do
```

Comment:
@margOnline you've done some good work on testing the sessions(_see below_). Is it possible to bring the sessions tests together so that they tell the stories about sessions in a single, consolidated fashion please?

```
context 'when editing details which have previously been submitted' do
        let(:session) do
          {
            age: 34,
            gender: 'female',
            salary: 34_125,
            salary_frequency: 'year',
            contribution_preference: 'minimum'
          }
        end

        let(:your_details_form) do
          assigns(:your_details_form)
        end

        before do
          get :new, nil, session
        end

        it 'sets age previously added' do
          expect(your_details_form.age).to eq(34)
        end

        it 'sets gender added' do
          expect(your_details_form.gender).to eq('female')
        end

        it 'sets salary previously added' do
          expect(your_details_form.salary).to eq(34_125)
        end

        it 'sets salary frequency previously added' do
          expect(your_details_form.salary_frequency).to eq('year')
        end

        it 'sets contribution preference previously added' do
          expect(your_details_form.contribution_preference).to eq('minimum')
        end
      end
```